### ISSUE TITLE
Display the number of images in a dataset

### ISSUE THREAD

**Issue by ManonGros** [gbif/portal-feedback#5512]
The idea came from an email sent by @dbloom. It would be nice to show the number of images in a dataset (not just the number of occurrences  associated with images).
This could be on the dataset page.

**Comment by dbloom** [comment]
Context: It is not uncommon for data publishers to ask me to confirm that all of their images/media have been published to GBIF. At the moment the only numbers present are the number of "occurrences published with images", but nowhere is the an actual count of images or media objects that have been published with the dataset unless a user reviews the metadata summary page for the resource in the IPT or if they download the DwC-A and look at the media file.

I would be a nice thing to have the number of images or media objects displayed someplace for reference for the publisher and also to aid discoverability for data users. I don't see this as a "need" to have, just a "nice" to have.

**Comment by dshorthouse** [comment]
Would it be workable to toggle on/off a dataset's StillImage filter in Occurrence search? See https://www.gbif.org/occurrence/search?dataset_key=f86a681d-7db8-483b-819a-248def18b70a&media_type=StillImage&advanced=1 by way of example.

**Comment by MortenHofft** [comment]
I do not believe the API supports this currently. I'm not even sure the underlying index really does either. I'm labelling this as a pipelines issue. But I might be wrong.

If we added a new field called e.g. `numberOfStillImages` we could do a sum of those I suppose. Doing it dynamically by filtering mediaItems would be awkward perhaps and possible require them to be indexed differently?

**Comment by dbloom** [comment]
All good questions @MortenHofft I had considered a tally of media objects, but to be honest nobody has ever asked me how many sound files or videos are available. Questions are always about how many images have been published. I am, however, open to other possibilities.
